pkgs/development/libraries/essentia/default.nix

```diff
@@ -0,0 +1,83 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, fetchpatch
+, libyaml
+, fftwFloat
+, ffmpeg
+, chromaprint
+, taglib
+, libsamplerate
+, pkgconfig
+# This one is not listed in upstream's instructions but it seems to be a
+# requirement since the build fails with the error:
+# [290/290] Linking build/src/libessentia.so
+# /nix/store/cl1i6bfqnx48ipakj4px7pb1babzs23j-binutils-2.31.1/bin/ld: cannot find -lQtCore
+# Probably because gaia which depends on qt4
+, qt4
+, wafHook
+, pythonSupport ? true
```

Comment:
Given that you also expose this via `pythonPackages`, doesn't it make sense to set this to `false` instead, so that Python support is left out when not explicitly using `pythonPackages.essentia`?

Reply:
You are right, I totally agree.